services/syllabus/routes.py

```diff
@@ -0,0 +1,131 @@
+##########################################################################
+# Name:     Users Routes
+# Purpose: File contains Syllabus request endpoints like create, update, fetch etc.
+#
+# Author:     Sai Nikhita Guduru
+#
+# Created:   07/07/2019
+# Copyright:   (c) Hoder Solutions Pvt Ltd 2018 - Present
+# Licence:   <your licence>
+##########################################################################
+from flask import Blueprint
+from flask import Response, request
+from json import dumps,loads
+from services.syllabus.models import Syllabus
+
+
+syllabus = Blueprint('syllabus', __name__)
+
+
+@syllabus.route("/syllabus", methods=["POST"])
+def api_add_syllabus():
+    syllabus = request.get_json()
+    syllabus_object = Syllabus(term = syllabus["term"], class_no = syllabus["class_no"], subject = syllabus["subject"], syllabus_details = dumps(syllabus["syllabus_details"]), syllabus_status = dumps(syllabus["syllabus_status"]))
```

Comment:
we need to follow PEP8 standards. the line has more than 80 characters, can you break it down into 2 lines